core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -193,8 +193,8 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
       }
 
       queryFuture.recover {
-        case dee:DataEntityException => RequestComplete(dee.code, dee.getMessage)
-        case ex => RequestComplete(StatusCodes.InternalServerError, ex)
+        case dee:DataEntityException => RequestComplete(ErrorReport(dee.code, dee.getMessage))
+        case ex => RequestComplete(ErrorReport(StatusCodes.InternalServerError, ex))
```

Comment:
wraps the bare strings in a json-serializable ErrorReport so clients (e.g. Terra UI) can machine-read them